contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -2171,7 +2171,12 @@ pltsql_store_func_default_positions(ObjectAddress address, List *parameters, con
 	}
 
 	if (!OidIsValid(get_bbf_function_ext_idx_oid()))
+	{	
+		pfree(func_signature);
```

Comment:
Should we also free default_positions?

Reply:
Noted it, will do that after discussion with Rishabh in different PR